crypto/x509/x509_vfy.c

```diff
@@ -2474,6 +2474,11 @@ void X509_STORE_CTX_set_flags(X509_STORE_CTX *ctx, unsigned long flags)
     X509_VERIFY_PARAM_set_flags(ctx->param, flags);
 }
 
+unsigned long X509_STORE_CTX_get_flags(X509_STORE_CTX *ctx)
+{
+    return X509_VERIFY_PARAM_get_flags(ctx->param);
+}
+
```

Comment:
This function is not strictly needed - one can simply combine `X509_STORE_CTX_get0_param()` and `X509_VERIFY_PARAM_get_flags()`.

Reply:
I wanted to keep the layers separate and similar so that if, as documented, stuff gets deprecated, it will be done cleanly.

Reply:
I meanwhile had a look at the deprecation note you referred to:

> The above functions should be used to manipulate verification parameters
> instead of functions which work in specific structures such as
> X509_STORE_CTX_set_flags() which are likely to be deprecated in a future release.

I read this as a point **not** to use (or even add) structure-specific functions like `X509_STORE_CTX_*_flags()`,
but for using the generic `X509_VERIFY_PARAM_*` functions, which are orthogonal to the the structures using verification flags.
So please not add to those functions that are likely to get deprecated.

Reply:
> So please not add to those functions that are likely to get deprecated.

It's an internal use.  It follows what the rest of that code does and is the least invasive change that could be made one week before the beta.
```
; g grep VERIFY_PARAM crypto/ocsp/
        exit 1
;
```

Reply:
No, you wanna add `X509_STORE_CTX_get_flags()` to the API (without documenting it),
and as mentioned such functions are likely to get deprecated and this function is not needed because
one can use `X509_STORE_CTX_get0_param()` followed by `X509_VERIFY_PARAM_get_flags()` instead.

Reply:
okay, fixup coming.  thanks.